.gitignore

```diff
@@ -92,6 +92,19 @@ fabric8-analytics-server/
 fabric8-analytics-jobs/
 fabric8-analytics-pgbouncer/
 fabric8-analytics-data-model/
+fabric8-analytics-stack-analysis/
```

Comment:
Do we expect people to have all those projects here?

Reply:
I am not sure. In parent directory, there is setup script which checkout all fabric8-analytics-* repositories.